packages/wrangler/src/__tests__/publish.test.ts

```diff
@@ -7386,6 +7386,46 @@ export default{
 					batch_size: 5,
 					max_retries: 10,
 					max_wait_time_ms: 3000,
+					max_concurrency: undefined,
+				},
+			});
+			await runWrangler("publish index.js");
+			expect(std.out).toMatchInlineSnapshot(`
+			"Total Upload: xx KiB / gzip: xx KiB
+			Uploaded test-name (TIMINGS)
+			Published test-name (TIMINGS)
+			  https://test-name.test-sub-domain.workers.dev
+			  Consumer for queue1
+			Current Deployment ID: Galaxy-Class"
+		`);
+		});
+
+		it("should support queue consumer concurrency with a max concurrency specified", async () => {
```

Comment:
Can we have a test where `null` is passed for `max_concurrency`?

Reply:
👍 Added!